docs/ide-usage.md

```diff
@@ -131,6 +135,9 @@ If you do not wish to use the automated setup, you can follow the [manual setup]
 
 1. To set up some static analysis tools, refer to [this document](static-analysis.md).
 
+1. To move on to the development phase, refer to [this document](development.md).
+
+
```

Comment:
Remove this blank line. The original file does not contain more than one blank line in a row.